sql/core/src/test/resources/sql-tests/results/explain.sql.out

```diff
@@ -65,48 +65,48 @@ struct<plan:string>
 
 
 (1) Scan parquet default.explain_temp1 
-Output [2]: [key#x, val#x]
+Output [2]: [key, val]
 Batched: true
 Location [not included in comparison]/{warehouse_dir}/explain_temp1]
 PushedFilters: [IsNotNull(key), GreaterThan(key,0)]
 ReadSchema: struct<key:int,val:int>
      
 (2) ColumnarToRow [codegen id : 1]
-Input [2]: [key#x, val#x]
+Input [2]: [key, val]
       
 (3) Filter [codegen id : 1]
-Input [2]: [key#x, val#x]
-Condition : (isnotnull(key#x) AND (key#x > 0))
+Input [2]: [key, val]
+Condition : (isnotnull(key) AND (key > 0))
      
 (4) Project [codegen id : 1]
-Output [2]: [key#x, val#x]
-Input [2]: [key#x, val#x]
+Output [2]: [key, val]
+Input [2]: [key, val]
      
 (5) HashAggregate [codegen id : 1]
-Input [2]: [key#x, val#x]
-Keys [1]: [key#x]
-Functions [1]: [partial_max(val#x)]
-Aggregate Attributes [1]: [max#x]
-Results [2]: [key#x, max#x]
+Input [2]: [key, val]
+Keys [1]: [key]
+Functions [1]: [partial_max(val)]
+Aggregate Attributes [1]: [max]
+Results [2]: [key, max]
      
 (6) Exchange 
-Input [2]: [key#x, max#x]
-Arguments: hashpartitioning(key#x, 4), true, [id=#x]
+Input [2]: [key, max]
+Arguments: hashpartitioning(key, 4), true, [id=#x]
       
 (7) HashAggregate [codegen id : 2]
-Input [2]: [key#x, max#x]
-Keys [1]: [key#x]
-Functions [1]: [max(val#x)]
-Aggregate Attributes [1]: [max(val#x)#x]
```

Comment:
It's still useful to have the attr id as the name can be duplicated
```
scala> sql("select 1 as a, 2 as a").explain
== Physical Plan ==
*(1) Project [1 AS a#53, 2 AS a#54]
+- Scan OneRowRelation[]
```

I think we should only remove the attr id from the auto-generated alias name. e.g. this should be `[max(val)#x]`

Reply:
I see. Thanks for the review :-)

The common modification in `AttributeReference` will also remove the `#` in `[max(val)#x]`. I'm trying to pin point to the exact creation time of auto-generated `AttributeReference` name.

Reply:
@cloud-fan I've re-implemented to only remove the auto-generated Attrids I think. Would you please help review again? Thanks so much!